articles/dd-you-did-what.md

```diff
@@ -0,0 +1,65 @@
+---
+authorName: Marko Ivanovic
+authorGithubUsername: ivma
+issue: 88
+title: You DiD what?
+---
+
+# {{page.title}}
+
+### Is it an outsourcing?! Is it a team sourcing?! No! It is the Distributed development!
+
+![DD image](./dd-you-did-what/dd-image.jpg)
+
+Dive into Distributed development topic and get all the information needed to kick off the project in Distributed Development Setup.
+
+### 1. What is it?
+Distributed development is a powerful way to realize project research, development, and realization across two or more physical locations. Difference between outsourcing and distributed development is that, in DD setup, all the organization is working together on the same level with a common goal of project realization. In Zuhlke Group, Distributed Development is a tool, a process, and a culture.
+
+### 2. How does it work?
+In the DD setup, there are multiple team members that are distributed to at least two or more locations. It's usual that team has a co-located project kick off, where all team members have an opportunity to meet each other in person, meet their Clients, stakeholder and get direct contact with all the key members of the project.
+
+The common language is English, for daily communication, meetings, documentation and relevant project resources.
```

Comment:
Consider removing this paragraph.

Putting the language here gives it more importance than the following points (transparency, collaboration etc). I would move it to the end of this subchapter but then, it will not add additional information (I mean, we collaborate with online tools, of course we speak a language both parties understand). 

In addition, the whole language aspect is discussed below in chapter 4 with much more added information. There it is a perfect fit.

Reply:
This was a good catch. I have modified it.